{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -19,8 +22,11 @@
 
 admin.site.site_title = '{{ cookiecutter.project_name }}'
 admin.site.site_header = '{{ cookiecutter.project_name }}'
-
+{%- if cookiecutter.multilingual == 'y' %}
+urlpatterns = i18n_patterns(
```

Comment:
There seems to be an issue with urls always redirecting to `/en/`. For example `/uni/` takes me to `/en/uni`.

Reply:
I think this is because of the language redirect in `pages.views`, it gets the language from the request, so might be that?